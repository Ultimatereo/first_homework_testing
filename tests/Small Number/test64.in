10
989 320 280 932 -699 847 -802 500 -413 864 -727
268 82 945 -945 900 343 -170 641 -950 409 -725
703 -202 -221 57 -765 -94 -897 -2 -520 -101 807
-67 -826 -154 -257 -109 -305 -855 571 -551 -527 192
-415 975 -513 912 -902 870 425 93 262 867 -857
-805 327 560 -89 -730 -609 337 -161 -460 109 -55
611 100 696 275 601 165 671 917 221 25 501
975 -929 873 -800 -957 470 -407 688 215 379 -127
-808 1 -344 168 -654 -621 -22 563 -178 126 101
337 104 -821 108 177 -701 18 -497 830 434 -71
