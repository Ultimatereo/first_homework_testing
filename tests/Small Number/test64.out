-0.554002
8.29972
2.4256
-2.616140
-0.733908
-0.805850
-6.09887
2.756738
5.953108
-5.45999